vue-frontend/src/components/contributions/WhatsTrending.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="tw-mt-5 xll:tw-container md:tw-mt-20">
+  <div class="tw-mt-5 xll:tw-container md:tw-mt-[225px]">
```

Comment:
use tailwind class for margin-top its available instead of pixel.Use rem for margins, padding, and font sizes